### PR TITLE
Adding Visual Studio debugging to Dev Quick Start Guide

### PR DESCRIPTION
**Pull request by ccaprio97**
## What’s changing?

Adding documentation to developer quick start guide for visual studio debugging and running

## Why?

NAC-1165

## This PR has:

- [ ] **Commented source code** (_required on public classes and methods, and elsewhere as appropriate_)

- [x] **Developer documentation** (_for new build/test/API changes or complex portions of the system_)

- [ ] **Automated unit tests** (_to maintain or increase level of code coverage_)

- [ ] **Changes to IAC** (_add any deployment steps that require manual intervention to the draft release notes_)

### CODE REVIEW

**Comment by ccaprio97** [comment]
@bgfoshay  this is first draft not expecting an actual review for approval but showing through git so the markdown appears clean

File: docs/quick-start-guide-dev.md [hunk 1]
```diff
@@ -203,6 +203,57 @@ Microsoft provides comprehensive [documentation on ARM templates](https://docs.m
 
 Once the IaC process has been run, individual applications are deployed to the resources using a [standardized build.bash script](../README.md#development) that can be run manually by a developer or as part of a CI/CD pipeline.
 
+## How to Run and Debug in Visual Studio
+
+Either create a json file called "launchSettings.json" in the properties folder of the function you're running or an "appsettings.json" file in the main project directory. 
+
+Variables will be pulled from the launchSettings.Json file if the developer uses **Environment.GetEnvironmentVariable** and the appsettings.json file if the developer uses **config[VARIABLE_NAME_HERE]**.
```
Comment (bgfoshay) [inline docs/quick-start-guide-dev.md:210]: Thought: For the appsettings.json scenario, mention that the IConfiguration interface should be used in order to access variables via config[VARIABLE_NAME_HERE]

Comment (ccaprio97) [reply docs/quick-start-guide-dev.md]: thoughts?

Comment (bgfoshay) [reply docs/quick-start-guide-dev.md]: 👍

File: docs/quick-start-guide-dev.md [hunk 2]
```diff
@@ -203,6 +203,57 @@ Microsoft provides comprehensive [documentation on ARM templates](https://docs.m
 
 Once the IaC process has been run, individual applications are deployed to the resources using a [standardized build.bash script](../README.md#development) that can be run manually by a developer or as part of a CI/CD pipeline.
 
+## How to Run and Debug in Visual Studio
+
+Either create a json file called "launchSettings.json" in the properties folder of the function you're running or an "appsettings.json" file in the main project directory. 
+
+Variables will be pulled from the launchSettings.Json file if the developer uses **Environment.GetEnvironmentVariable** and the appsettings.json file if the developer uses **config[VARIABLE_NAME_HERE]**. When using from the appsettings.json the IConfiguration interface should be used.
```
Comment (dborgen2) [inline docs/quick-start-guide-dev.md:210]: Would it be helpful at all to make a note here on why, as a developer, you should choose one or the other when developing your code? Or is that out of scope of this file?

Comment (ccaprio97) [reply docs/quick-start-guide-dev.md:210]: I'm leaning towards out of the scope of this but we could have @bgfoshay  put his 2 cents in

**Review by dborgen2 — APPROVED** [review]
👍

**Pull request merged**